openmdao/components/interp_util/tests/test_interp_nd.py

```diff
@@ -56,60 +57,24 @@ def test_interp_spline_akima_derivs(self):
 
```

Comment:
I am nervous about losing this original test of the derivative values.  I would copy this to a new test (put it in the Feature test object since it is in the docs) and then revert the changes here back to the original.

Reply:
I'll add the original test to the non features test class. I don't think it's necessary for the user to see a huge output chunk